### PR TITLE
remove vsa & ceph references - bsc#1076227

### PR DESCRIPTION
**Pull request by csymons-suse**
separate commits for vsa and ceph edits

### CODE REVIEW

File: xml/releasenotes50.xml [hunk 1]
```diff
@@ -19,7 +19,9 @@
  <xi:include href="newfeatures50.xml"/>
  <xi:include href="fixedissues50.xml"/>
  <xi:include href="knownissues50.xml"/>
- <xi:include href="unsupported_features.xml"/>
+ <!-- csymons 2018-05-25:
+     <xi:include href="unsupported_features.xml"/>
+-->
```
Comment (sknorr) [inline xml/releasenotes50.xml]: It is unlikely that we will ever ship the HOS 5 release notes again -- what is the point of this change?

File: xml/operations-storage_alarmdefinitions.xml [hunk 2]
```diff
@@ -1155,7 +1157,7 @@ ansible-playbook -i hosts/verb_hosts cinder-migrate-volume.yml</screen>
          Use the Ceph start playbook against the affected node:
         </para>
 <screen>cd ~/scratch/ansible/next/ardana/ansible
-ansible-playbook -i hosts/verb_hosts ceph-start.yml --limit &lt;hostname&gt;</screen>
+ansible-playbook -i hosts/verb_hosts ceph-start.yml dashdashlimit &lt;hostname&gt;</screen>
```
Comment (sknorr) [inline xml/operations-storage_alarmdefinitions.xml]: Please do `-/-` instead. (That is much more standard.)

File: xml/architecture-third_party-third_party_service_integration.xml [hunk 3]
```diff
@@ -848,9 +847,7 @@ hooks.d/&lt;custom&gt;/post-&lt;action&gt;.yml</screen>
      <filename>nova-deploy.yml</filename> Typically, these hook playbooks simply
      call out to other third-party playbooks, e.g.:
     </para>
-<screen>stack@hlm:~/scratch/ansible/next/ardana/ansible$ cat hooks.d/myservice/post-nova-deploy.yml
-#
-# (c) Copyright 2017 Hewlett Packard Enterprise Development Company LP
+<screen>ardana@ardana:~/scratch/ansible/next/ardana/ansible$ cat hooks.d/myservice/post-nova-deploy.yml
```
Comment (sknorr) [inline xml/architecture-third_party-third_party_service_integration.xml]: Same as the comment from #282 -- either correct or just remove this prompt.

File: xml/esx-remove_existing_cluster_compute_resource_pool.xml [hunk 4]
```diff
@@ -242,21 +242,24 @@ ansible-playbook -i hosts/verb_hosts neutron-stop --limit &lt;hostname&gt;;</scr
...
 +--------------------------------------+--------------------------------------+-----------------------+-------------------+-------------------------------------------+----------+-------+-----------------+------+--------------------------+--------------------------+--------------------------+
 | cfc6bfa4-2485-4319-b1e5-0107886f4270 | cca96c53-a927-4b0a-9bf3-cb21d28216f3 | Host Status           | host_alive_status | service: system                           | HIGH     | OK    | None            | None | 2016-10-27T06:33:04.256Z | 2016-10-27T06:33:04.256Z | 2016-10-23T13:41:57.258Z |
-|                                      |                                      |                       |                   | cloud_name: entry-scale-kvm-esx-vsa-mml   |          |       |                 |      |                          |                          |                          |
+|                                      |                                      |
+|                   | cloud_name: entry-scale-kvm-esx-mml       |          |       |                 |      |                          |                          |                          |
```
Comment (sknorr) [inline xml/esx-remove_existing_cluster_compute_resource_pool.xml]: These tables seems [even] broken[er] now. Are these line breaks accidental?

File: xml/releasenotes50.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/releasenotes50.xml]: Reverted.
I guess that this deletion and all places where it is used should just be removed. No need for releasenotes50.xml RN501 or RN502. They are included in introduction.xml, which is included in book.hos-index.xml.  Can I remove all of them?

File: xml/esx-remove_existing_cluster_compute_resource_pool.xml (hunk 4 above)
Comment (csymons-suse) [reply xml/esx-remove_existing_cluster_compute_resource_pool.xml]: fixed. Yes they were accidental...or rather part of the intentional, helpful nature of emacs ensuring 78 character lines.

**Pull request merged**